lib/fpm/package/virtualenv.rb

```diff
@@ -72,9 +76,14 @@ def input(package)
                "pip", "distribute")
     safesystem(pip_exe, "uninstall", "-y", "distribute")
 
-    safesystem(pip_exe, "install", "-i",
-               attributes[:virtualenv_pypi],
-               package)
+    extra_index_url_args = []
+    if !attributes[:virtualenv_pypi_extra_index_urls].nil?
```

Comment:
Suggestion - instead of checking !foo.nil?, you can just check foo:

```
# if !attributes[:virtualenv_pypi_extra_index_urls].nil?
if attributes[:virtualenv_pypi_extra_index_urls]
```

Non-nil values are "truthy" in ruby. Just an FYI, no code change required.